src/test/java/org/apache/commons/io/HexDumpTest.java

```diff
@@ -253,6 +253,16 @@ public void testDumpOutputStream() throws IOException {
 
         // verify proper behavior with null stream
         assertThrows(NullPointerException.class, () -> HexDump.dump(testArray, 0x10000000, null, 0));
+
```

Comment:
How about reusing `ThrowOnCloseInputStream`?

Reply:
@garydgregory Thanks for the pointer. I was not aware of this test utility class. Code updated.